zerver/lib/bugdown/__init__.py

```diff
@@ -622,7 +636,7 @@ def run(self, root):
                       class_attr=class_attr)
                 continue
             if self.is_image(url):
-                add_a(root, url, url, title=text)
+                add_a(root, self.get_actual_image_url(url), url, title=text)
```

Comment:
I would add an `if` statement similar to the `dropbox_image` one above.  

Also, we need to make sure that we don't translate actual images on GitHub's site, like, this:

https://developer.github.com/assets/images/hero-circuit-bg.svg

Can you add a test for that case?

Probably part of our check for whether we need to do translate should be that `/blob/` appears in the URL in the right place.